src/utils/accounts.js

```diff
@@ -17,67 +17,82 @@
     with this program; if not, write to the Free Software Foundation, Inc.,
     51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 */
-import get from 'lodash.get';
 import type { Account, Accounts, AccountTypes } from 'models/Account';
 import { ACCOUNT_TYPES } from 'constants/accountsConstants';
 import { addressesEqual } from './assets';
 
-export function getActiveAccount(accounts: Accounts = []): ?Account {
+export const getActiveAccount = (accounts: Accounts): ?Account => {
   return accounts.find(({ isActive }) => isActive);
-}
+};
 
-export function getActiveAccountId(accounts: Accounts = []): string {
-  const activeAccount = getActiveAccount(accounts) || {};
-  return activeAccount.id || '';
-}
+export const getInactiveUserAccounts = (accounts: Accounts): Accounts => {
+  return accounts.filter(({ isActive }) => !isActive);
+};
 
-export function getActiveAccountType(accounts: Accounts = []): string {
-  const activeAccount = getActiveAccount(accounts) || {};
-  return activeAccount.type || '';
-}
+export const getActiveAccountId = (accounts: Accounts): string => {
+  const activeAccount = getActiveAccount(accounts);
+  if (!activeAccount) {
```

Comment:
What if simple no brackets return?

Reply:
Hmm.. Well this seems a bit subjective, and I understand there could be some personal differences.

That being said, I think one liners move the code towards more dense and less legible syntax. The more things we have on each line, the more difficult it becomes to read a given block of code.

Lines are not free, but they are cheaper than complexity. Plus, if a file gets too big, we can move parts of it into other files. While if the code gets too complex, we have to refactor it, which is harder to do and harder to validate everything's still working as expected. This is not gonna be a huge difference on one way or another, but I think it's gonna be a bit more consistent with more complex functions, which should be a good thing.

Personally, I feel like this:

```js
const func = (a?: number): number => {
  if (!a) {
    return 0;
  }

  return a / 2;
}
```

Is cleaner than:

```js
const func = (a?: number): number => {
  if (!a) return 0;
  return a / 2;
}
```

Of course there are some exceptions, but normally I wouldn't try to squeeze things on less lines.

What do you think?

Reply:
as an old-school dev I agree with @grilix, although in modern books you'll find that code readability is not important our days, and it's normal to write the code this way: `func = () => () => () => {}`. But good luck to construct this in your head when you do a quick code reading